.github/workflows/e2e.yaml

```diff
@@ -108,3 +131,65 @@ jobs:
       - name: Collect troubleshooting data
         uses: ./.github/actions/collect-troubleshooting
         if: ${{ always() }}
+
+  e2e-hub:
+
+    runs-on: ubuntu-latest
+    name: e2e-hub
+    needs: [build-operator-image, build-operator-hub-image, build-signing-image]
+
+    services:
+      registry:
+        image: registry:2
+        ports: ['5000:5000/tcp']
+
+    steps:
+
+      - name: Install curl on the machine
+        run: sudo apt update -y && sudo apt upgrade -y && sudo apt install -y curl
```

Comment:
Why is this needed?

Reply:
We need it in order to install the `clusteradm` command in the e2e script:
```
# Install the `clusteradm` command
curl -L https://raw.githubusercontent.com/open-cluster-management-io/clusteradm/main/install.sh | bash
```